### PR TITLE
enhancement: Enable reply box for whatsapp template message 

### LINKED ISSUE #2081: Enable reply box for whatsapp template message after 24 hour reply window
Currently the twillio whatsapp support template messages. But the reply box is hidden due to the 24-hour reply restriction.

cc @sojan-official

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes #2081 
**Preview**
reply

### CODE REVIEW

File: app/javascript/dashboard/i18n/locale/en/conversation.json [hunk 1]
```diff
@@ -20,6 +20,8 @@
     "LOADING_CONVERSATIONS": "Loading Conversations",
     "CANNOT_REPLY": "You cannot reply due to",
     "24_HOURS_WINDOW": "24 hour message window restriction",
+    "TWILLIO_WHATSAPP_CAN_REPLY": "You can reply on template message due to",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/conversation.json:23]: "You can only reply to this conversation using a template message due to"

File: app/javascript/shared/constants/links.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const REPLY_POLICY = {
+  FACEBOOK:
+    'https://developers.facebook.com/docs/messenger-platform/policy/policy-overview/',
+  TWILLIO_WHATSAPP:
```
Comment (pranavrajs) [inline app/javascript/shared/constants/links.js:4]: `TWILIO`, spelling mistake

File: app/javascript/dashboard/i18n/locale/en/conversation.json [hunk 3]
```diff
@@ -20,6 +20,8 @@
...
+    "TWILLIO_WHATSAPP_24_HOURS_WINDOW": "24 hour message window restriction",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/conversation.json:24]: spelling mistake `twilio`

**Comment by muhsin-k** [comment]
@pranavrajs Please check the latest changes

File: app/javascript/dashboard/components/widgets/conversation/MessagesView.vue [hunk 4]
```diff
@@ -191,6 +211,12 @@ export default {
       }
       return '';
     },
+    facebookReplyPolicy() {
+      return REPLY_POLICY.FACEBOOK;
+    },
+    twilloWhatsAppReplyPolicy() {
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/MessagesView.vue:217]: `twilio`

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please see the individual comment.

**Review by pranavrajs — APPROVED** [review]

**Comment by muhsin-k** [comment]
**QA**
- Configure Twilio WhatsApp inbox
- Send a WhatsApp message to the newly added number 
- Wait for 24 hours
- Send template message (ex: "Your appointment is coming up on {{1}} at {{2}}") from agent dashboard
- Check message is delivered successfully

**Review by sojan-official — APPROVED** [review]

**Pull request merged**